modules/logging-content-filter-drop-records.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-content-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-content-filter-drop-records_{context}"]
+= Configuring content filters to drop unwanted log records
+
+When content filters are configured, the log collector edits log streams according to the filters before forwarding. If you configure the `drop` filter, the collector drops unwanted log records that match the specified configuration.
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+* You have created a `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+. Add configuration for a filter to the `filters` spec in the `ClusterLogForwarder` CR.
++
+The following example shows how to configure the `ClusterLogForwarder` CR to drop log records based on regular expressions:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  filters:
+  - name: <filter_name>
+    type: drop # <1>
+    drop: # <2>
+      tests: # <3>
+      - field: # <4>
+        matches:
+        notMatches:
+  pipelines:
+  - name: <pipeline_name> # <5>
+      filterRefs: ["<filter_name>"]
+# ...
+----
+<1> Specify the type of filter. The `drop` filter completely drops log records that match the filter configuration.
+<2> Specify configuration options for applying the `drop` filter.
+<3> Specify the configuration for tests that are used to evaluate whether a log record is dropped. If all the conditions specified for a test are true, the test passes and the log record is dropped. When multiple tests are specified for the `drop` filter configuration, if any of the tests pass, the record is dropped. If there is an error evaluating a condition, for example, the field is missing from the log record being evaluated, that condition evaluates to false.
+<4> Specify a JSON path to the field. The `field` stanza requires at least one of the following conditions:
++
+matches:: Specify a regular expression. If log records match this regular expression, they are dropped.
```

Comment:
@Clee2691 please confirm... you can only specify one or the other, but not both?

Reply:
Yes only one of `matches` or `notMatches` can be defined at once for drop filters NOT both

Reply:
Thanks for clarifying! I will edit the docs to make this explicit.